miniapp/USERGUIDE.md

```diff
@@ -209,7 +209,7 @@ miniAppMessageBridge.setUserInfoBridgeDispatcher(userInfoBridgeDispatcher)
 
 ### #5 Create and display a Mini App
 
-Calling `MiniApp.create` with a Mini App ID object will download the latest version of the Mini App if it has not yet been downloaded. A view will then be returned which will display the Mini App.
+Calling `MiniApp.create` with a Mini App ID or `MiniAppInfo` object will download the latest version of the Mini App if it has not yet been downloaded. A view will then be returned which will display the Mini App.
```

Comment:
I think this isn't true, is it? Using the Mini App ID will download the latest version, but using `MiniAppInfo` will download the specific version in the info.

Reply:
Sorry, let me edit it. I didn't read the rest of doc 😆.